debug-scripts/network

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -ux
+
+ifconfig -a > $DEBUG_SCRIPT_DIR/ifconfig
+cp /etc/resolv.conf $DEBUG_SCRIPT_DIR/resolv.conf
+cp /etc/network/interfaces $DEBUG_SCRIPT_DIR/interfaces
+netstat -planut > $DEBUG_SCRIPT_DIR/netstat
+dig google.com> $DEBUG_SCRIPT_DIR/dig-google
```

Comment:
AH this may be troublesome behind egress limited firewalls.... but thats also good information to have.